### PR TITLE
Documentation update - towards full SI

### PR DESCRIPTION
**Pull request by LDAmorim**
Hi, this PR contains only two minor changes. I am making it a PR just so you can correct me if I am changing the docs incorrectly.
I am using sphinx installed with conda with `sphinx-build 2.1.2`.
For the html to work without warnings / errors I had to make the following changes:

- Added blank line after list
- Changed characters in link to Q. H. Liu paper from (`>` to `\>`, for example).

I still see the warning message:
```/Users/diana/Documents/opt/WarpX/Docs/source/running_cpp/pwfa.rst: WARNING: document isn't included in any toctree```
Is this issue because the file is not yet integrated?

If you agree with the two small changes, I can go ahead and change the documentation according to issue #290, with small commits / PRs like this one.
Thanks, Diana

### CODE REVIEW

**Comment by MaxThevenet** [comment]
This file is actually not included anywhere. If you find it useful, you can add it at the end of `running_cpp/running_cpp.rst` and reformat it a little bit (see for instance `running_cpp/how_to_run.rst` for an example). Basically avoid long lines and replace 
```
In-depth explanation of a PWFA example
--------------------------------------
```
with
```
In-depth explanation of a PWFA example
======================================
```

**Comment by LDAmorim** [comment]
Ok, I will do that.
Thanks!

**Comment by MaxThevenet** [comment]
The file looks slightly incomplete though (does not point to an input file, for instance), so there is probably some more work to do to make it really useful to the doc.

**Comment by LDAmorim** [comment]
Yes, I am thinking the same.
In that case, here are the small changes and the order in which I am thinking of doing them:
1 - change the other files so that they respect the CONTRIBUTING.md instruction of:
`The number of characters per line should be <80`
2 - make the changes for SI in all the other files.
3 - prepare the PWFA section based on the other ones (probably linking to the pwfa example files) 
What do you think?

**Comment by MaxThevenet** [comment]
Sounds great

**Comment by ax3l** [comment]
Just a note on the number of chars per line for non-code files: I usally treat `.md` file docs and `.rst` file docs like LaTeX sources. One sentence per line creates the smallest `diff`s & least frustration during editing and edits soft-wrap them anyway.

**Comment by RemiLehe** [comment]
Also: @LDAmorim I think you will need to merge `dev` into your branch for the automated tests to pass.

**Comment by LDAmorim** [comment]
Yes, I will do that.

**Comment by MaxThevenet** [comment]
Is this work in progress? If so, could we prepend the name of the PR with tag `[WIP]`? Thanks

**Comment by dpgrote** [comment]
@LDAmorim What is the status of this work? I can approve the several small changes here if this is still needed so this can be merged.

**Comment by LDAmorim** [comment]
Hi @dpgrote! Sorry, I will have to re-check this PR for what I still need to do in it. But I will try to update you on it's status asap.

**Comment by LDAmorim** [comment]
Hi all,

Sorry for the huge delay in the progress of this PR.

For context, I've been updating and including the file `running_cpp/pwfa.rst`.
That file contains a nice description of a typical/realistic (aka SLAC inspired setup) particle beam driven plasma wakefield acceleration configuration.
This, I think, is important to include in WarpX as it is the ECP project application developed to model plasma accelerators.

I am probably making conversion mistakes, but it seems to me that the scenario the file describes does not match the configuration of any of the pwfa examples of the WarpX repo.

I think it would be interesting to adapt it to describe either an example input file, such as 

> warpx/Examples/Physics_applications/plasma_acceleration/inputs_2d_boost

 or our milestone-like single plasma acceleration stage setup (which most of us resort to to make tests on performance and accuracy).

Either one could even be complemented with an output illustrative image.

I also think that this wouldn't take much additional time from me.

What do you think?

Thanks,
Diana

**Comment by MaxThevenet** [comment]
Sorry for the late reply. Yes we can adapt the text to match the PWFA example, but already having just a description of PWFA, even if the parameters don't match, would be much better than what we currently have.

**Comment by LDAmorim** [comment]
Hi @MaxThevenet !
Thanks for checking this. I have been and am working on it. Plan to get this done very soon.
If you want, I can ping you once I get the draft of the doc page ready.

**Comment by MaxThevenet** [comment]
Sure, or you can request a review by @dpgrote or myself when the PR is ready.

**Comment by MaxThevenet** [comment]
Could finalize and merge this PR? When another review is needed, you can remove the [WIP] tag and request one from me.

**Comment by LDAmorim** [comment]
Thanks @MaxThevenet ! I am working on it and I will do that :)

**Comment by LDAmorim** [comment]
Thanks for all the patience with this PR @MaxThevenet , @RemiLehe and @dpgrote ,

I just finished the draft of the PWFA example description.
It now includes also the output of the example simulation:
PWFA

I did it thinking that the people who would most benefit from this description are future new WarpX users/inexperienced developers.
Who can then get familiar with the code by running the example and by looking at the output.
 
If you have time, I would very much like to ask you for your feedback (that can be very general, so I fix the main direction of the document if you want) again.

ps: I thought in the example the beam was not being defocused... should I play with the setup as well to "fix" that?

Thanks

**Comment by MaxThevenet** [comment]
Is this ready for review? If so, could you remove the [WIP] tag?

**Comment by LDAmorim** [comment]
Sorry @MaxThevenet ! I removed it

File: Docs/source/visualization/advanced.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ electric and magnetic fields are all defined at different locations in space. Re
 returns cell-centered data for convenience, which involves an additional operation. It is sometimes
 useful to access the raw data directly. Furthermore,
 the WarpX implementation for mesh refinement contains a number of grids for each level (coarse,
-fine and auxiliary, see
-:doc:`../theory/warpx_theory` for more details), and it is sometimes useful to access each of
+fine and auxilary, see
```
Comment (MaxThevenet) [inline Docs/source/visualization/advanced.rst]:
```suggestion
fine and auxiliary, see
```

Comment (LDAmorim) [reply Docs/source/visualization/advanced.rst]: Oh sorry for this @MaxThevenet !

**Review by MaxThevenet — APPROVED** [review]
Thanks for this PR! having one very detailed explanation is probably going to be helpful for new users, so that we can refer to it even for slightly different runs.

**Pull request merged**